### DISCUSSION TITLE
Issue templates now use Issue forms instead of markdown

### DISCUSSION THREAD

**Discussion by Ayush7614** [fossology/FOSSologyUI#189] [General]
Markdown templates often made it difficult to capture the whole context, by expecting the contributors to read the markup, uncomment lines and delete relevant context.

Issue forms beta was released for public repositories last month, as mentioned by GitHub on this [blog](https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/). We would like to migrate the issue templates to YAML-based forms which are easy to fill up and maintain, helping us capture relevant context.

There is no issue templates in the project so i would like to work on the issue

**Comment by Ayush7614** [comment]
cc @Shruti3004 @GMishx

**Reply by GMishx** [reply]
From #184:

In my opinion, it will be premature to move to issue forms right now.
But if you feel there will be additional benefit from it (current issue templates are confusing), please go ahead and open a PR.
